bridge_adaptivity/module/forms.py

```diff
@@ -27,7 +27,6 @@ class GroupForm(ModelForm):
         choices=GRADING_POLICY_CHOICES,
         required=True,
     )
-    course = forms.ModelChoiceField(queryset=Course.objects.all())
```

Comment:
I would leave this field in the form with the assumption it could left blank.

Reply:
it's not needed, because `course` field in `CollectionGroup` model has `blank=True, null=True`